docs/src/semilinear/Linear.md

```diff
@@ -0,0 +1,48 @@
+# OrdinaryDiffEqLinear
+
+Methods for semi-linear differential equations.
+
+```@eval
+first_steps = evalfile("./common_first_steps.jl")
+first_steps("OrdinaryDiffEqLinear", "LieRK4")
```

Comment:
The typical example is probably not appropriate for linear solvers.

Reply:
There is an example in the docs right now IIRC that you can copy.